src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="page">
+      <div class="calendar calendar--start-day--thu calendar--month-length--29">
```

Comment:
there's no class for idicating the calendar is the element of the page

Reply:
It's not necessary in this case.